pkg/common/config/types.go

```diff
@@ -89,4 +93,10 @@ type VirtualCenterConfig struct {
 	CAFile string `gcfg:"ca-file"`
 	// Thumbprint of the VCenter's certificate thumbprint
 	Thumbprint string `gcfg:"thumbprint"`
+	// IP Family enables the ability to support IPv4, IPv6, or a dual
+	// stack with priority given to a certain IP type ipv6,ipv4
+	// Default: ipv4
+	IPFamily string `gcfg:"ip-family"`
+	// IPFamilyPriority the list/priority of IP versions
+	IPFamilyPriority []string
```

Comment:
Hi @dvonthenen,

Is this missing a `gcfg` tag?

Reply:
It's on purpose. IPFamily contains the user-provided string... where IPFamilyPriorty is just a convenient place holder that attempts to parse the user-provided input into something that's more usable for me.